spharpy/samplings/samplings.py

```diff
@@ -153,14 +153,15 @@ def hyperinterpolation(n_points=None, n_max=None, radius=1.):
         sep=' ').reshape((int(n_points), 4))
 
     # normalize weights
```

Comment:
```suggestion
    # normalize weights to 4pi
```
maybe also add in the docstring that the weights are normalized to 4pi

Reply:
and fix the doc in general SamplingSphere is returned and we can also mention about weather its a quadrature or not

Reply:
added a note to the docstring of all quadrature samplings.